.meteor/packages

```diff
@@ -51,7 +51,6 @@ rocketchat:favico
 rocketchat:file
 rocketchat:github-enterprise
 rocketchat:gitlab
-rocketchat:highlight
```

Comment:
please, add this package back. this package does code highlight which we don't want to remove :smile: